execute/profiler.go

```diff
@@ -0,0 +1,64 @@
+package execute
+
+import (
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/memory"
+	"github.com/influxdata/flux/values"
+)
+
+type Profiler interface {
+	Name() string
+	GetResult(q flux.Query, alloc *memory.Allocator) (flux.Table, error)
+}
+
+var AllProfilers map[string]Profiler = make(map[string]Profiler)
+
+func RegisterProfilers(ps ...Profiler) {
+	for _, p := range ps {
+		AllProfilers[p.Name()] = p
+	}
+}
+
+type FluxStatisticsProfiler struct{}
+
+func init() {
+	RegisterProfilers(FluxStatisticsProfiler{})
+}
+
+func (s FluxStatisticsProfiler) Name() string {
+	return "FluxStatistics"
+}
+
+func (s FluxStatisticsProfiler) GetResult(q flux.Query, alloc *memory.Allocator) (flux.Table, error) {
+	groupKey := NewGroupKey(
+		[]flux.ColMeta{
+			{
+				Label: "_measurement",
+				Type:  flux.TString,
+			},
+		},
+		[]values.Value{
+			values.NewString("profiler/FluxStatistics"),
+		},
+	)
+	b := NewColListTableBuilder(groupKey, alloc)
+	for _, colName := range []string{"_measurement", "_field", DefaultValueColLabel} {
```

Comment:
Do we want to use the `_field/_value` schema or the pivoted schema where each statistic is its own column.

Is see below that all stats are considered strings. I think that makes working with the stats data too cumbersome.

If we used a wide schema with each stat being its own column that would allow for each column to have its own type.

Thoughts?